bundler/lib/bundler/current_ruby.rb

```diff
@@ -65,20 +66,24 @@ def truffleruby?
       RUBY_ENGINE == "truffleruby"
     end
 
-    def mswin?
+    def windows?
       Gem.win_platform?
     end
 
+    def mswin?
+      windows? && Bundler.local_platform != Gem::Platform::RUBY && Bundler.local_platform.os == "mswin32" && Bundler.local_platform.cpu == "x86"
+    end
```

Comment:
Note that the meaning of the `mswin?` method is now changing to mean specifically local_platform `mswin32`. For all Windows, should now use the `windows?` method. I doubt this will affect downstream gems, but just FYI.

Reply:
Since it's potentially backwards incompatible, I'd rather roll it back for now, yeah. Can be discussed separately later.

Reply:
OK, rolled back.